src/Services/Air/templates/AIR_LOW_FARE_SEARCH_REQUEST.handlebars.js

```diff
@@ -4,12 +4,21 @@ module.exports = `
 <!--Air Low Fare Search For Galileo({{provider}}) Request-->
 <soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
     <soap:Body>
+        {{#if async}}
+        <air:LowFareSearchAsynchReq
+            AuthorizedBy="user" TraceId="{{requestId}}" TargetBranch="{{TargetBranch}}"
```

Comment:
This part seems duplicated below. Isn't enough just to change an opening tag in case of the async method?

Reply:
Maybe, but wouldn't it be more clear to understand  if they're in separate lines?

Do you prefer 
```
<air:{{#if async}}LowFareSearchAsynchReq{{#else}}LowFareSearchReq{{/if}}
```

rather than this?
```
{{#if async}}
<air:LowFareSearchAsynchReq ...
{{#else}}
<air:LowFareSearchReq ...
{{/if}}
```